Tests/pens/ttGlyphPen_test.py

```diff
@@ -638,6 +638,96 @@ def test_cubic(self):
                     ("closePath", ()),
                 ]
 
+    def test_cubic_topology(self):
+        for preserveTopology in (False, True):
```

Comment:
this looks like could be parametrized with `pytest.mark.parametrize` decorator

Reply:
parametrizing both the input and the expected output

Reply:
While it's possible, it would have 8 lengthy parameters. I'm not sure it's a good idea. WDUT?

Reply:
I'm trying it.

Reply:
only 3 parameters, preserve_topology, expected_coordinates, expected_flags

Reply:
Done.

Reply:
thanks! that's much better